GitUI/CommandsDialogs/BrowseDialog/FormBisect.cs

```diff
@@ -48,11 +49,15 @@ private void UpdateButtonsState()
 
         private void Start_Click(object sender, EventArgs e)
         {
-            FormProcess.ShowDialog(this, GitCommandHelpers.StartBisectCmd());
+            bool success = FormProcess.ShowDialog(this, process: null, arguments: GitCommandHelpers.StartBisectCmd(), Module.WorkingDir, input: null, useDialogSettings: true);
+            if (!success)
+            {
+                return;
+            }
+
             UpdateButtonsState();
```

Comment:
We're starting a bisect operation here, if `bisect start` has failed (i.e. `success = false`) there's no point to update buttons. Don't you think?

Reply:
Git operations are *not* atomic. The user could also try to abort a command. (I had to do this several times when my mouse double clicked instead of single by itself in the left panel. It ended up with some of the files from the new branch and the old one kept active.)
Don't put too much logic into UI handlers in order to save a little runtime. Rather keep it simple and safe.

Reply:
reverted